doc: config_and_build: Note on how to pass Kconfig to child image (#14110)
nrfconnect/sdk-nrf: doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

Simultaneous multi-image DFU
****************************

> .. note::

Reviewer: We don't usually have "Notes" immediately after a heading title (I could be wrong about this?)
I would suggest to move this to  after line 594.

Or, if this info is very important to have it here in the beginning I would take it out of the "note".

Reviewer 2: This suggestion is OK, but now we have two notes one after another, which sometimes is a bit of an overkill.

Author: > This suggestion is OK, but now we have two notes one after another, which sometimes is a bit of an overkill.

Yeah I agree. I wonder, whether instead of the note there should be rather some introduction chapter, on the page, that would remind on how the Kconfig options work with child-images and how to address child images, instead of doing remark here.

It kinda reminds me what Larry Trusk has written about footnotes in "Punctuation" (https://www.sussex.ac.uk/informatics/punctuation/essaysandletters/footnotes); in short:
```
 There are two main rules in the use of footnotes. First:

    Do not use a footnote if you can possibly avoid it. 
```

Reviewer 2: That is a good point, as too many notes lead to users ignoring them. I don't oppose the idea of the introduction, but in this particular case I also think that the ``.. note::`` is not needed and this information can leave as normal text in the new location after the list of Kconfigs.

Author: OK, I will remove the `.. note::` and make it part of text.

>    As described in :ref:`multi-image build guide <ug_multi_image_variables>`, make sure to the child image name prefix to the name of the Kconfig options that are used when building MCUboot as child image (for example, ``child_image_CONFIG_PCD_APP``.

Reviewer: ```suggestion
   As described in :ref:`multi-image build guide <ug_multi_image_variables>`, make sure to name the child image prefix to the name of the Kconfig options that are used when building MCUboot as a child image (for example, ``child_image_CONFIG_PCD_APP``.
```

Reviewer 3: maybe: "to add the child image prefix (...)"?

>    If you don't do this, the MCUboot will be built as the standard child image with the ``mcuboot`` prefix (for example, ``mcuboot_CONFIG_PCD_APP``).

Reviewer: ```suggestion
   If you don't do this, MCUboot will be built as the standard child image with the ``mcuboot`` prefix (for example, ``mcuboot_CONFIG_PCD_APP``).
```

Simultaneous multi-image DFU
****************************

.. note::

>    When building MCUboot as child image, within a multi-image project, the Kconfig options provided
>    via command line or general project file need to be prefixed with a child image name.
>    If a user has not changed anything the MCUboot will be built as mcuboot child image
>    and the prefix will be mcuboot_, for example mcuboot_CONFIG_PCD_APP.

Reviewer 2: ```suggestion
   As described in :ref:`multi-image build guide <ug_multi_image_variables>`, make sure to the child image name prefix to the name of the Kconfig options that are used when building MCUboot as child image (for example, ``child_image_CONFIG_PCD_APP``.
   If you don't do this, the MCUboot will be built as the standard child image with the ``mcuboot`` prefix (for example, ``mcuboot_CONFIG_PCD_APP``).
```
